### PR TITLE
Added image preprocessing

### PR DESCRIPTION
**Pull request by aASDa213ASD**
For #7

### CODE REVIEW

File: src/preprocessor.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+"""Requires: opencv-python, numpy"""
+class Preprocessor:
+    def __init__(
+        self,
+        contrast: float, invert_colors: bool,
+        denoise_intensity: int,
+        sharpen_intensity: float,
+        input_path: str, output_path: str = None
+    ):
```
Comment (Zarathustra4) [inline src/preprocessor.py:14]: Can you add some default values?

File: src/preprocessor.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            [-1, self.sharpen_intensity, -1],
+            [-1, -1, -1]
+        ])
+        img = cv.filter2D(img, 0, kernel)
+
+        """ 
+            Warning: Do not use, sh*t
+            Morphological operations to remove noise and fill gaps
+        """
+        #kernel = np.ones((2, 2), np.uint8)
```
Comment (Zarathustra4) [inline src/preprocessor.py:48]: Do we need the commented lines or we can use them depending on some input parameter?

Comment (aASDa213ASD) [reply src/preprocessor.py:48]: You can play with those, makes letter way thicker, might help in some rare occasions

File: src/preprocessor.py (hunk 1 above)
Comment (aASDa213ASD) [reply src/preprocessor.py:14]: Sure, pushing soon

Comment (aASDa213ASD) [reply src/preprocessor.py:14]: Added [default parameters](https://github.com/AIVMZB/Latina/commit/7ea01c57763b1fdb168e4feae90bc198e61b13cc), as you asked

**Pull request merged**